cluster_config.tf

```diff
@@ -95,6 +95,7 @@ resource "aws_s3_bucket_object" "configurations" {
       hive_metsatore_username             = var.metadata_store_adg_writer_username
       hive_metastore_pwd                  = jsondecode(data.aws_secretsmanager_secret_version.rds_aurora_secrets.secret_string)["password"]
       hive_metastore_endpoint             = aws_rds_cluster.hive_metastore.endpoint
+      hive_metastore_database_name        = aws_rds_cluster.hive_metastore.database_name
```

Comment:
Nice; could we add the database name as an output to https://github.com/dwp/aws-analytical-dataset-generation/blob/adg-point-to-hive-metsatore/metastore.tf#L125-L130 as well please, so that other consumers know what it's called? Ta.

Reply:
Thanks Matt , added it to outputs